### PR TITLE
chore: update golang to 1.19

### PR DESCRIPTION
**Pull request by navoiee**
This PR updates go and packages to latest versions and fixes lint issues.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
   test:
     docker:
-      - image: cimg/go:1.14.1
+      - image: cimg/go:1.19
+        environment:
+          GOFLAGS: -p=8
```
Comment (joegrimeringresso) [inline .circleci/config.yml:7]:
```	-p n
		the number of programs, such as build commands or
		test binaries, that can be run in parallel.
		The default is GOMAXPROCS, normally the number of CPUs available.
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
     steps:
       - checkout
       - run:
-          name: reinstall certs package
-          command: sudo apt-get update && sudo apt-get install --reinstall ca-certificates  
+          name: Avoid hosts unknown for github
+          command: echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
```
Comment (joegrimeringresso) [inline .circleci/config.yml:12]: Shady stuff

**Review by joegrimeringresso — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (navoiee) [reply .circleci/config.yml:12]: Maybe but we are using this everywhere ;)

File: .circleci/config.yml (hunk 1 above)
Comment (navoiee) [reply .circleci/config.yml:7]: Do you think we should change it? I think the circleci will be fine.

**Pull request merged**